scout/constants/filters.py

```diff
@@ -23,3 +23,10 @@
     "region_annotations": ["exonic", "splicing"],
     "functional_annotations": SEVERE_SO_TERMS,
 }
+
+CLINICAL_FILTER_BASE_MEI = {
+    "variant_type": "clinical",
+    "region_annotations": ["exonic", "splicing"],
+    "functional_annotations": SEVERE_SO_TERMS_SV,
```

Comment:
Modelling this on SVs to avoid issues with vep annotation, but all variants I've seen from retroseq so far have been represented as clean insertions at one single base position.